cloudify/manager.py

```diff
@@ -356,6 +356,8 @@ def get_resource(blueprint_id, deployment_id, tenant_name, resource_path):
             blueprint_id, deployment_id, tenant_name, resource_path):
         try:
             return get_resource_from_manager(path)
+        except NonRecoverableError:
```

Comment:
i realize `except NonRecoverable` is ugly, because it's supposed to be "non recoverable" :) but this is the only backwards-compatible way for both functions (get_resource and get_resource_from_manager), and both are kindasorta public...